src/standalone/Bucket/Bucket.jl

```diff
@@ -65,51 +65,60 @@ abstract type AbstractBucketModel{FT} <: AbstractExpModel{FT} end
 abstract type AbstractLandAlbedoModel{FT <: AbstractFloat} end
 
 """
-    BulkAlbedoFunction{FT, F <: FUnction} <: AbstractLandAlbedoModel
+    BulkAlbedoFunction{FT, F <: ClimaCore.Fields.Field} <: AbstractLandAlbedoModel
 
 An albedo model where the albedo of different surface types
 is specified. Snow albedo is treated as constant across snow
-location and across wavelength. Surface albedo (sfc)
+location and across wavelength. Bareground surface albedo
 is specified as a function
 of latitude and longitude, but is also treated as constant across
-wavelength; surface is this context refers to soil and vegetation.
+wavelength; bareground surface is this context refers to soil and vegetation.
 """
-struct BulkAlbedoFunction{FT, F <: Function} <: AbstractLandAlbedoModel{FT}
+struct BulkAlbedoFunction{FT, F <: ClimaCore.Fields.Field} <:
+       AbstractLandAlbedoModel{FT}
     α_snow::FT
-    α_sfc::F
+    α_bareground::F
+end
+
+function BulkAlbedoFunction{FT}(
+    α_snow::FT,
+    α_bareground::F,
+) where {FT <: AbstractFloat, F}
+    BulkAlbedoFunction(α_snow, α_bareground)
 end
 
 function BulkAlbedoFunction{FT}(
     α_snow::FT,
     α_sfc::Function,
```

Comment:
maybe we should also call this alpha_bareground for clarity?

Reply:
i guess we cant quite call it that, but maybe something similar instead of `sfc`

Reply:
good point! what about `alpha_bareground_func`?